modules/form-field/react/stories/stories_Radio.tsx

```diff
@@ -14,142 +13,233 @@ const hintId = 'error-desc-id';
 storiesOf('Components|Inputs/Radio/React/Top Label/Radio Group', module)
   .addParameters({component: RadioGroup})
   .addDecorator(withReadme(README))
-  .add('Default', () => (
-    <FormField label="Label" useFieldset={true}>
-      <ControlledComponentWrapper>
-        <RadioGroup name="contact">
-          <Radio id="1" value="email" label="E-mail" />
-          <Radio id="2" value="phone" label="Phone" />
-          <Radio id="3" value="fax" label="Fax (disabled)" disabled={true} />
-          <Radio id="4" value="mail" label="Mail" />
-        </RadioGroup>
-      </ControlledComponentWrapper>
-    </FormField>
-  ))
-  .add('Alert', () => (
-    <FormField
-      label="Label"
-      useFieldset={true}
-      error={FormField.ErrorType.Alert}
-      hintText={hintText}
-      hintId={hintId}
-    >
-      <ControlledComponentWrapper>
-        <RadioGroup name="contact">
-          <Radio id="1" value="email" label="E-mail" />
-          <Radio id="2" value="phone" label="Phone" />
-          <Radio id="3" value="fax" label="Fax (disabled)" disabled={true} />
-          <Radio id="4" value="mail" label="Mail" />
-        </RadioGroup>
-      </ControlledComponentWrapper>
-    </FormField>
-  ))
-  .add('Error with Grow', () => (
-    <FormField
-      label="Label"
-      useFieldset={true}
-      error={FormField.ErrorType.Error}
-      grow={true}
-      hintText={hintText}
-      hintId={hintId}
-    >
-      <ControlledComponentWrapper>
-        <RadioGroup name="contact">
-          <Radio id="1" value="email" label="E-mail" />
-          <Radio id="2" value="phone" label="Phone" />
-          <Radio id="3" value="fax" label="Fax (disabled)" disabled={true} />
-          <Radio id="4" value="mail" label="Mail" />
-        </RadioGroup>
-      </ControlledComponentWrapper>
-    </FormField>
-  ));
+  .add(
+    'Default',
+    () => {
+      const [value, setValue] = React.useState(undefined);
+      return (
+        <FormField label="Label" useFieldset={true}>
+          <RadioGroup name="contact" value={value} onChange={value => setValue(value)}>
+            <Radio id="1" value="email" label="E-mail" />
+            <Radio id="2" value="phone" label="Phone" />
+            <Radio id="3" value="fax" label="Fax (disabled)" disabled={true} />
+            <Radio id="4" value="mail" label="Mail" />
+          </RadioGroup>
+        </FormField>
+      );
+    },
+    {
+      chromatic: {
```

Comment:
Also, since we're trying to limit the number of testing stories, I think we need to be more intentional about how we test this (rather than just turning chromatic on for each existing story). We will likely need to consider other input components inside of FormField as well

Reply:
Yep. Good call. Will update.